snap/component_test.go

```diff
@@ -55,6 +55,7 @@ type: test
 version: 1.0
 summary: short description
 description: long description
+provenance: prov
```

Comment:
do we still have tests here where this is empty but validates?

Reply:
Yes, `TestReadComponentInfoMinimal` covers that.